main/settings.js

```diff
@@ -70,6 +70,24 @@ function save() {
     fs.writeFileSync(config.getSettingsJsonPath(), JSON.stringify(data));
 }
 
+function loadSources() {
+    if (sourcesData !== null) {
+        return;
+    }
+    let sources = parseJSONFile(config.getSourcesJsonPath());
```

Comment:
How about s/parseJSONFile/parseJsonFile since all the other places are written as 'Json'